README.md

```diff
@@ -35,21 +35,23 @@ workaround:
 
 ## How to install *realpath_turbo*
 
-1. Download the *realpath_turbo* source code.
-
-2. Extract and compile the extension
-
-   ```sh
-   $ tar -xaf realpath_turbo*
-   $ cd realpath_turbo*
-   $ phpize
-   $ ./configure
-   $ make
-   # make install
-   ```
-
-3. Adjust your `php.ini` to load and configure *realpath_turbo* extension.
-
+```sh
+wget -qO- https://github.com/Whissi/realpath_turbo/archive/master.tar.gz | tar -xz
+cd realpath_turbo-master/
+
+phpize
+./configure
+make
+make test NO_INTERACTION=1; echo $?

```

Comment:
Adding `make test` to the README is a good idea and follow best practice.
I would skip the `echo` because you will see a textual output showing an error. I don't think people understand a single "0" or non zero value.